docroot/themes/humsci/humsci_basic/src/js/main-nav/menu-toggle.js

```diff
@@ -0,0 +1,56 @@
+const menuToggle = document.querySelector('.su-main-nav__toggle');
+const mainMenu = document.querySelector('.su-main-nav__menu-lv1');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+let isHidden;
+let wasDesktopSize;
+
+const collapseNav = () => {
+  menuToggle.setAttribute('aria-expanded', false);
+  mainMenu.setAttribute('aria-hidden', true);
+  menuToggle.innerHTML = "Menu";
```

Comment:
we actually dont have to set the text at all here. If we keep it menu in the menu block code template, the that content can be translated by drupal if we ever need. Also, a screenreader user will hear, with the aria-expanded attribute: Menu Button Expanded / Menu button collapsed and that's all they really need to know!